api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -32,11 +32,13 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol with HmdaCust
   val parseLarRoute =
     pathPrefix("lar") {
       path("parse") {
-        timedPost {
-          entity(as[String]) { s =>
-            LarCsvParser(s) match {
-              case Right(lar) => complete(ToResponseMarshallable(lar))
-              case Left(errors) => complete(errorsAsResponse(errors))
+        hmdaAuthorize {
```

Comment:
Do we want to require authorization for `/lar/parse`?  Seems like we want anonymous access to this resource.